config/policies/lockdown-profilebrowser.json

```diff
@@ -0,0 +1,8 @@
+{
+    "IncognitoModeAvailability": 1,
+    "TorDisabled": true,
```

Comment:
Only question about this one, maybe we want to leave option to use Tor for future use? I guess could also change.